nms/app/theme/design-system/Text.tsx

```diff
@@ -71,10 +68,10 @@ type Props = {
     | 'body2'
     | 'body3'
     | 'caption'
-    | 'overline',
-  className?: string,
-  useEllipsis?: ?boolean,
-  weight?: 'inherit' | 'light' | 'regular' | 'medium' | 'bold',
+    | 'overline';
+  className?: string;
+  useEllipsis?: boolean | null | undefined;
```

Comment:
Nit pick having both `?` and `| undefined` is somewhat redundant in this context